solidity/test/bridge/Bridge.Governance.test.ts

```diff
@@ -960,6 +960,145 @@ describe("Bridge - Governance", () => {
     )
   })
 
+  describe("beginRedemptionTxMaxTotalFeeUpdate", () => {
+    context("when the caller is not the owner", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(thirdParty)
+            .beginRedemptionTxMaxTotalFeeUpdate(1)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when the caller is the owner", () => {
+      let tx: ContractTransaction
+
+      before(async () => {
+        await createSnapshot()
+
+        tx = await bridgeGovernance
+          .connect(governance)
+          .beginRedemptionTxMaxTotalFeeUpdate(1337)
+      })
+
+      after(async () => {
+        await restoreSnapshot()
+      })
+
+      it("should not update the redemption tx max total fee", async () => {
+        const { redemptionTxMaxTotalFee } = await bridge.redemptionParameters()
+        expect(redemptionTxMaxTotalFee).to.be.equal(
+          constants.redemptionTxMaxTotalFee
+        )
+      })
+
+      it("should start the redemption tx max total fee timer", async () => {
+        const blockTimestamp = (await ethers.provider.getBlock(tx.blockNumber))
+          .timestamp
+        const initTimestamp = (await tx.wait()).events[0].args.timestamp
+        const elapsedTime = BigNumber.from(blockTimestamp).sub(initTimestamp)
+
+        expect(
+          BigNumber.from(constants.governanceDelay).sub(elapsedTime)
+        ).to.be.equal(constants.governanceDelay)
+      })
+
+      it("should emit RedemptionTxMaxTotalFeeUpdateStarted event", async () => {
+        const blockTimestamp = (await ethers.provider.getBlock(tx.blockNumber))
+          .timestamp
```

Comment:
`await helpers.time.lastBlockTime()` ? We can fix all occurrences in a separate PR, should be an easy find & replace.

Reply:
https://github.com/keep-network/tbtc-v2/pull/378